concepts/eventing-code-guidelines.md

```diff
@@ -75,11 +54,6 @@ For these reasons, `uber-go/zap` is the **preferred** structured logging library
 
 Furthermore, consider using `github.com/kyma-project/kyma/common/logging/logger`, which provides further **abstraction** over `uber-go/zap`.
 
-<!-- voting options:
-1. [ueber-go/zap](https://github.com/uber-go/zap) // proposed by guidelines
```

Comment:
Do we need a line stating the logger we must use?

Reply:
We do already have that, check [structured logging](https://github.com/kyma-project/community/blob/4ebb59628325f2acc626e02e94dc28a02853e837/concepts/eventing-code-guidelines.md#structured-logging) section :)

Reply:
Ah I missed it! All good 👍